arches_querysets/models.py

```diff
@@ -58,6 +58,9 @@ def serialize(self, **kwargs):
             for key, val in vars(self).items()
         }
 
+    def __iter__(self):
+        return iter(vars(self).items())
```

Comment:
Are you worried at all that this will imply that order is preserved somehow?

You could get a deceiving looking consistency from python dicts preserving insertion order, but keys in the tile are unordered.

Reply:
I'll let @johnatawnclementawn speak to his own use case here (I don't expect he's relying on ordering) - but it's true I don't want to set an expectation that this is ordered.  I'm not sure what the alternative is though, @jacobtylerwalls

Reply:
I'm not relying on ordering in my use case (ETL Lingo resources into SKOS RDF), but if we're worried about returning an iterable that _looks_ ordered, we could enforce ordering with something like this
```suggestion
    def __iter__(self):
        return iter(dict(sorted(vars(self).items())))
```

Reply:
Ah, well I wouldn't want to introduce a notion of ordering :D

I guess I'm still worried this is not intuitive. AliasedData is like a plain dict right now, so I'd expect iterating a dict to give me keys. `if "appellative_status" in aliased_data"` etc. Now I'd have to do `if "appellative_status" in aliased_data.keys()`. Have you poked around to see if that `in aliased_data` syntax is in use anywhere?

Reply:
> AliasedData is like a plain dict right now, so I'd expect iterating a dict to give me keys. if "appellative_status" in aliased_data" etc. 

It's not actually behaving like a dict in this way though:
`[key for key in aliased_data]` or `"appellative_status" in aliased_data`
yields:
`TypeError: 'AliasedData' object is not iterable`

In fact, it seems that we would need to implement `keys()` in order for `if "appellative_status" in aliased_data.keys()` to return anything other than `AttributeError`. Furthermore, since SimpleNamespace is configured for attribute-based access, we'd have to implement these methods for AliasedData to behave like a plain dict.
```suggestion
   def __iter__(self):
        return iter(self._keys())

    def _keys(self):
        return vars(self).keys()

    def _values(self):
        return vars(self).values()

    def _items(self):
        return vars(self).items()
```
I opted to for internal use naming convention in case there are node aliases with those values - open to suggestions here.

As for poking around for the `in aliased_data` syntax, I don't see anything that's doing that. (here's regex I used: `(?<=in)(.*)(?=aliased_data)`, limited to .py files)
I do see one instance of drilling into a TileTree like this in the [Lingo serializers](https://github.com/archesproject/arches-lingo/blob/0726b1bddf410c2d526c8ec0717eb84a5ed135f0/arches_lingo/serializers.py#L44), maybe you're thinking of that?

Reply:
Ah right of course. Your plan makes sense. Adding a dunder method seems fine, as I think we disallow __ in node aliases somewhere.

Resolving!